alerts.md

```diff
@@ -16,6 +16,10 @@ Kubecost alerts allow teams to receive updates on real-time Kubernetes spend. Th
  
 6. [Cluster Health](#type-cluster-health) -- used to determine if the cluster's health score changes by a specific threshold.
 
+7. [Asset Budget](#type-asset-budget) -- sends an email and/or Slack alert when a particular asset spend crosses a defined threshold.
+
+8. [Cloud Report](#type-cloud-report) -- sends an email and/or Slack alert with asset spend across all or a subset of cloud resources.
+
 Have questions or issues? View our [troubleshooting guide](#troubleshooting).
   
 ## Configuring Alerts in Helm
```

Comment:
Can we change all H2 and lower headers to sentence case? "Configuring alerts in Helm", "Asset budget", etc.

Reply:
Don't you think it's better with Heading here? As a reader i can navigate to type-asset-budget when i click on line Number #19. If there's a valid reason to change it to sentences I'm okay with it.

Reply:
Yeah keep the headers as they are, I just wouldn't capitalize every word in the headers besides the title of the page.